Source/lighting.cpp

```diff
@@ -82,12 +82,11 @@ std::vector<Displacement> CrawlRow(int row)
  *    +-------> x
  */
 
-const std::vector<std::vector<Displacement>> CrawlTable = [] {
-	std::vector<std::vector<Displacement>> ret;
-	for (int i = 0; i < 19; i++)
-		ret.push_back(CrawlRow(i));
-	return ret;
-}();
+const std::array<std::vector<DisplacementOf<int8_t>>, MaxCrawlRow + 1> CrawlTable {
+	std::vector<DisplacementOf<int8_t>> { { static_cast<int8_t>(0), static_cast<int8_t>(0) } },
+	CrawlRow(1), CrawlRow(2), CrawlRow(3), CrawlRow(4), CrawlRow(5), CrawlRow(6), CrawlRow(7), CrawlRow(8), CrawlRow(9),
+	CrawlRow(10), CrawlRow(11), CrawlRow(12), CrawlRow(13), CrawlRow(14), CrawlRow(15), CrawlRow(16), CrawlRow(17), CrawlRow(18)
+};

```

Comment:
Are we porting to BIOSs now or why are we unrolling this loop?

Reply:
Fixed